domains/ffmuc_freising.conf

```diff
@@ -51,6 +51,7 @@
       enabled = 'true',
       iface = 'wg_mesh_vpn',
       limit = '1', -- actually unused
+      broker = 'broker.ffmuc.net',
```

Comment:
This should also be 'broker.ffmuc.net/api/v1/wg/key/exchange', right?

Reply:
Yep